cpp/python/pystrings.cpp

```diff
@@ -135,6 +136,52 @@ bool parse_args( const char* fn, PyObject* pyargs, const char* pyfmt, ... )
     return rtn;
 }
 
+static std::basic_string<char> ConvertCudaIpcMemHandler (cudaIpcMemHandle_t ipc_memhandle) {
+    std::basic_string<char> bytes;
+    bytes.resize(sizeof(cudaIpcMemHandle_t));
+    memcpy((void*)bytes.data(), (char*)(&ipc_memhandle), sizeof(cudaIpcMemHandle_t));
+    return bytes;
+}
+
+static cudaIpcMemHandle_t ConvertByteArray (PyObject* bytearray) {
+    char* string_array = PyByteArray_AsString(bytearray);
+    cudaIpcMemHandle_t ipc_memhandle;
+    memcpy((char*)&ipc_memhandle, string_array, sizeof(cudaIpcMemHandle_t));
+    return ipc_memhandle;
+}
+
+static PyObject* n_createFromIPC( PyObject* self, PyObject* args )
```

Comment:
This does not look right. You are reading values from the args as though they are passed in as separate parameters. From the companion method below, it looks like it should be reading from the list.
And from the .py methods, no parameters are actually being passed in to either method.

Reply:
Thank you, it was fixed.